src/RapidezStatamicServiceProvider.php

```diff
@@ -199,6 +203,31 @@ public function bootPublishables() : self
         return $this;
     }
 
+    public function bootUserModel() : static
+    {
+        if (!class_exists('\App\Models\User')) {
+            config(['auth.providers.users.model' => User::class]);
+        }
+
+        $migrationPath = database_path('migrations/0001_01_01_000000_create_users_table.php');
+
+        if (!file_exists($migrationPath)) {
+            try {
+                $userTable = file_get_contents('https://raw.githubusercontent.com/laravel/laravel/refs/heads/11.x/database/migrations/0001_01_01_000000_create_users_table.php');
+
+                if (!$userTable) {
+                    throw new Exception('Failed to download the migration file.');
```

Comment:
So if laravel moves this file, we'll get exceptions on every production site that includes this code? 😲

Reply:
No, as this will be caught and logged in the catch.
Also will only happen if there is no existing migration, which should never be on a production environment.